### ISSUE TITLE
Reference: Known bugs

### ISSUE THREAD

**Issue by kaarleol** [nowcommunity/nowdatabase#392]
#233 related

Check discord/reference

**Comment by Deeroil** [comment]
From Discord, roughly translated:

"- journal can't be deleted
- Author delete button doesn't always work right. If you try to delete a author from a latter author field, wrong author gets deleted (the one with the earlier order number?)
- Reference deleting button doesn't do anything

Both probably need edits/refactoring in EditableTable, SelectingTable and EditingForm. At least SelectingTable has to be changed, maybe others start working then as well"

**Comment by kaarleol** [comment]
Delete button is its own task, since backend doesn't handle deleting authors & journals correctly when deleting a reference so really that should be done first (as of 28.10)

Author bug: 
Not exactly. There are multiple author fields per reference.

If you add 2 authors on each field, then try to delete a an author from the second field, the corresponding author from the first field will be deleted, not from the second one.

Example:
Field1
 Au1
 Au2

Field2
 Au3
 Au4
 Au5

If you try to delete Au4, Au2 will be deleted instead
If you try to delete Au3, Au1 will be deleted instead
If you try to delete Au5, Au3 will be deleted instead
Deleting from the first field works correctly
